Rosetta workflow failed due to fixed typo in configuration file

### Description

`rosetta-cli` release v0.7.3 fixed a typo in its configuration file thus our github wokflow failed at rosetta-cli loading its configuration

### Steps to reproduce

```
coinbase/rosetta-cli info checking GitHub for latest tag
Installing Rosetta CLI
coinbase/rosetta-cli info found version: 0.7.3 for v0.7.3/linux/amd64
coinbase/rosetta-cli info installed ./rosetta-cli
Rosetta CLI already installed. Skipping installation
Running Rosetta data API Validation against testnet Network
2021/12/08 22:08:16 json: unknown field "inactive_discrepency_search_disabled": unable to unmarshal: unable to open configuration file: unable to load configuration
Failed to Pass data API Validation
Rosetta CLI already installed. Skipping installation
Running Rosetta construction API Validation against testnet Network
2021/12/08 22:08:16 json: unknown field "inactive_discrepency_search_disabled": unable to unmarshal: unable to open configuration file: unable to load configuration
Failed to Pass construction API Validation
Error: Process completed with exit code 1.
```

### Additional context

_No response_

### Hedera network

testnet

### Version

v0.46.0-SNAPSHOT

### Operating system

_No response_

--- a/hedera-mirror-rosetta/scripts/validation/demo/validation.json
+++ b/hedera-mirror-rosetta/scripts/validation/demo/validation.json
@@ -13,7 +13,7 @@
   "tip_delay": 300,
   "data": {
     "reconciliation_disabled": false,
-    "inactive_discrepency_search_disabled": false,
+    "inactive_discrepancy_search_disabled": false,
     "balance_tracking_disabled": false,
     "bootstrap_balances": "./data_genesis_balances.json",
     "end_conditions": {

--- a/hedera-mirror-rosetta/scripts/validation/testnet/validation.json
+++ b/hedera-mirror-rosetta/scripts/validation/testnet/validation.json
@@ -25,7 +25,7 @@
   },
   "data": {
     "reconciliation_disabled": false,
-    "inactive_discrepency_search_disabled": false,
+    "inactive_discrepancy_search_disabled": false,
     "balance_tracking_disabled": false,
     "bootstrap_balances": "./data_genesis_balances.json",
     "end_conditions": {

--- a/hedera-mirror-rosetta/test/bdd-client/README.md
+++ b/hedera-mirror-rosetta/test/bdd-client/README.md
@@ -88,6 +88,6 @@ The following table lists the available properties along with their default valu
 | `hedera.mirror.rosetta.test.server.dataRetry.max`       | 20                    | The max retries of a data request                                          |
 | `hedera.mirror.rosetta.test.server.offlineUrl`          | http://localhost:5701 | The url of the offline rosetta server                                      |
 | `hedera.mirror.rosetta.test.server.onlineUrl`           | http://localhost:5700 | The url of the online rosetta server                                       |
-| `hedera.mirror.rosetta.test.server.httpTimeout`         | 15s                   | The timeout of an http request sent to the rosetta server                  |
+| `hedera.mirror.rosetta.test.server.httpTimeout`         | 25s                   | The timeout of an http request sent to the rosetta server                  |
 | `hedera.mirror.rosetta.test.server.submitRetry.backOff` | 200ms                 | The amount of time to wait between submit request retries                  |
 | `hedera.mirror.rosetta.test.server.submitRetry.max`     | 5                     | The max retries of a submit request                                        |

--- a/hedera-mirror-rosetta/test/bdd-client/config.go
+++ b/hedera-mirror-rosetta/test/bdd-client/config.go
@@ -52,7 +52,7 @@ hedera:
              max: 20
            offlineUrl: http://localhost:5701
            onlineUrl: http://localhost:5700
-           httpTimeout: 15s
+           httpTimeout: 25s
            submitRetry:
              backOff: 200ms
              max: 5

